packages/apps-v5/src/error_info.js

```diff
@@ -109,11 +109,6 @@ module.exports = [
     'title': 'Blank app',
     'level': 'info'
   },
-  {
-    'name': 'H82',
-    'title': 'Free dyno quota exhausted',
```

Comment:
Instead of removing, should this be updated to the new H82 error instead? https://devcenter.heroku.com/articles/error-codes#h82-you-ve-used-up-your-dyno-hour-pool

Reply:
👍